BUILD.gn

```diff
@@ -0,0 +1,191 @@
+# Copyright 2019 Google Inc. All rights reserved.
```

Comment:
Can we use the standard OpenThread license header?

Reply:
Sure, will fix this.